SPECS-EXTENDED/perl-Digest-BubbleBabble/perl-Digest-BubbleBabble.spec

```
@@ -0,0 +1,224 @@
+Summary:        Create bubble-babble fingerprints
+Name:           perl-Digest-BubbleBabble
+Version:        0.02
+Release:        29%{?dist}
+License:        GPL+ OR Artistic
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://metacpan.org/release/Digest-BubbleBabble
+Source0:        https://cpan.metacpan.org/modules/by-module/Digest/Digest-BubbleBabble-%{version}.tar.gz#/perl-Digest-BubbleBabble-%{version}.tar.gz
```

```suggestion
Source0:        https://cpan.metacpan.org/modules/by-module/Digest/Digest-BubbleBabble-%{version}.tar.gz#/%{name}-%{version}.tar.gz
```

If that's a common thing in the other specs, this will be my last suggestion comment.:) We could update these in bulk.